### PR TITLE
Drop dex

### PR DESCRIPTION
**Pull request by krasish**
**Description**

This PR removes the usage of dex in the compass console replacing it with a configurable OIDC provider. 

Changes proposed in this pull request:

- Remove dex references
- Update README.md

### CODE REVIEW

**Review by nkanchev — DISMISSED** [review]

File: compass/package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@asyncapi/converter": "^0.6.1",
+    "@asyncapi/converter": "^0.6.0",
```
Comment (alextargov) [inline compass/package.json]: Why do we degrade this package?

Comment (krasish) [reply compass/package.json]: Fixed

**Review by nkanchev — APPROVED** [review]

**Review by alextargov — APPROVED** [review]

**Pull request merged**